README.md

```diff
@@ -30,16 +30,15 @@ async function logMessagesFromFooBar() {
 
   await bag.open();
 
-  // read all messages from both the '/foo' and '/bar' topics:
-  await bag.readMessages({ topics: ["/foo", "/bar"] }, (result) => {
+  for await (const result of bag.forwardIterator({ topics: ["/foo", "/bar"] })) {
```

Comment:
I like the method name `readMessages` better than `forwardIterator`. `readMessages` and `readMessagesReverse`?

Reply:
`readMessages` sounds too much like "read all the messages and return them" rather than "get me a cursor to read the messages". I agree tho that `forwardIterator` is maybe too programatic, tho it is worth noting that the returned instance _is_ actually an iterator (i.e. has a symbol iterator). I will go with `messageIterator` and use `{reverse: true}` as an option to get a reverse message iterator.